pyrobosim/examples/demo_prm.py

```diff
@@ -15,15 +15,21 @@
 
 def test_prm():
     """Creates a PRM planner and plans"""
-    prm = PRMPlanner(world, max_nodes=100, max_connection_dist=1.5)
+    planner_config = {
```

Comment:
This demo works, but the graph colors and line styles changed since `main`, and in my opinion they look a lot worse and harder to visualize. Consider using the same "gentler" semi-transparent blue as original.

Also, I'm not sure what's happening but this demo runs very slowly.

Reply:
The slowness is probably due to the the call to `show_path_and_planner()` rather than just `show_path()`.
Since it redraws the graph every time navigate is clicked.  I was supposed to change that but forgot. Will change it.